Examples/GroupSync/Azure_Group_Sync.ps1

```diff
@@ -132,15 +128,15 @@ if (-not $Session) {
 	exit;
 }
 
-ForEach($region in $regions) {
-	if ($region.Name -notmatch $RegionsMatch) {
+ForEach($region in $response.values) {
+	if ($region.name -notmatch $RegionsMatch) {
 		Continue;
 	}
 
-	$GroupName = $GroupPrefix + "_" + $region.Name;
+	$GroupName = $GroupPrefix + "_" + $region.name;
 	Write-Verbose "Processing $GroupName";
 
-	$region.IpRange.Subnet |
+	$region.properties.addressPrefixes | Select-String -not ':' |
```

Comment:
Yes, new JSON list includes IPv6. I just filtered here for a fast implementation as I was not interested in IPv6